roofit/xroofit/src/xRooBrowser.cxx

```diff
@@ -61,8 +61,8 @@ xRooBrowser::xRooBrowser(xRooNode *o) : TBrowser("RooBrowser", o, "RooFit Browse
 
    // override file menu event handling so that can intercept "Open"
    if (auto rb = dynamic_cast<TRootBrowser *>(GetBrowserImp())) {
-      rb->Disconnect(rb->fMenuFile, "Activated(Int_t)", rb, "HandleMenu(Int_t)");
-      rb->fMenuFile->Connect("Activated(Int_t)", ClassName(), this, "HandleMenu(Int_t)");
+      rb->Disconnect(GETPOPUPMENU(rb,fMenuFile), "Activated(Int_t)", rb, "HandleMenu(Int_t)");
+      GETPOPUPMENU(rb,fMenuFile)->Connect("Activated(Int_t)", ClassName(), this, "HandleMenu(Int_t)");
```

Comment:
```suggestion
      TGPopupMenu * popupMenu = static_cast<TRootBrowserInterface *>(rb)->GetMenuFile();
      rb->Disconnect(popupMenu, "Activated(Int_t)", rb, "HandleMenu(Int_t)");
      popupMenu->Connect("Activated(Int_t)", ClassName(), this, "HandleMenu(Int_t)");
```
Can you try please if this would work then?

Reply:
https://stackoverflow.com/questions/57451850/casting-an-object-to-a-derived-class-to-access-protected-members-of-parent-class

Reply:
Alright I take back my suggestion then!